src/core/index.js

```diff
@@ -430,6 +436,13 @@ export function captcha(m) {
   return get(m, 'captcha');
 }
 
+export function passwordlessCaptcha(m) {
+  if (typeof m !== 'object') {
```

Comment:
Is this check necessary?  Should get handle non-objects (just thinking of lodash)?

Reply:
I copied it from the `captcha()` function above, but it looks like this isn't needed for the current tests. 🤔 For now I removed it from both, but maybe @stevehobbsdev can double check this.